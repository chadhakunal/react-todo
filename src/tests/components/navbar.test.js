import { shallow } from "enzyme";
import Navbar from "components/navbar";

describe('Navbar', () => {
  it('should render correctly with no props', () => {
    const component = shallow(< Navbar />);
    expect(component).toMatchSnapshot();
  });
})