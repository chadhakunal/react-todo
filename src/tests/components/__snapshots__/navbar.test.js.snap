// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navbar should render correctly with no props 1`] = `
<div>
  <Navbar
    className="navbar-bordered"
    dark={true}
    expand="md"
    tag="nav"
  >
    <div
      className="container pt-2 pb-1"
    >
      <NavbarBrand
        href="/"
        id="navbar-brand"
        tag="a"
      >
        Todo App
      </NavbarBrand>
    </div>
  </Navbar>
</div>
`;
